cherrypy/process/servers.py

```diff
@@ -182,13 +175,26 @@ def start(self):
 
         self.wait()
         self.running = True
-        self.bus.log('Serving on %s' % on_what)
+        self.bus.log('Serving on %s' % self.description)
     start.priority = 75
 
+    @property
+    def description(self):
```

Comment:
Sounds like a premature optimization, especially when the only known invocation only gets invoked once per start call.